neqo-transport/src/recovery/sent.rs

```diff
@@ -61,41 +62,54 @@ impl SentPacket {
     }
 
     /// The type of this packet.
+    #[must_use]
     pub const fn packet_type(&self) -> PacketType {
         self.pt
     }
 
     /// The number of the packet.
+    #[must_use]
     pub const fn pn(&self) -> PacketNumber {
         self.pn
     }
 
     /// The ECN mark of the packet.
+    #[must_use]
     pub const fn ecn_mark(&self) -> IpTosEcn {
         self.ecn_mark
     }
 
     /// The time that this packet was sent.
+    #[must_use]
     pub const fn time_sent(&self) -> Instant {
         self.time_sent
     }
 
     /// Returns `true` if the packet will elicit an ACK.
+    #[must_use]
     pub const fn ack_eliciting(&self) -> bool {
         self.ack_eliciting
     }
 
     /// Returns `true` if the packet was sent on the primary path.
+    #[must_use]
     pub const fn on_primary_path(&self) -> bool {
         self.primary_path
     }
 
     /// The length of the packet that was sent.
+    #[must_use]
     pub const fn len(&self) -> usize {
         self.len
     }
 
+    #[must_use]
+    pub const fn is_empty(&self) -> bool {
+        self.len() == 0
+    }
```

Comment:
Why did you add this?  Packets cannot be empty, so it doesn't make sense to have this.  You should suppress the warning instead.

Reply:
Yes. Sorry about that. Fixed with https://github.com/mozilla/neqo/pull/2253/commits/1ed1ca09e6dccb7685c3b88db2afde943247d379.